### DISCUSSION TITLE
Using Gauge --> Prometheus 'of_meter_*' metrics

### DISCUSSION THREAD

**Discussion by dnlygvtv** [faucetsdn/faucet#3912] [Q&A]
Hello guys, first I would like to thank you for your great work on this project! 💪

I am a newbie in SDN and your Faucet-controller but I try to go through it by myself. I have a question about how I can use 'of_meter_*' metrics, imported in Prometheus by Gauge.
In controller source code I found that Gauge don't log this data into InfluxDB (method _GaugeMeterStatsInfluxDBLogger()_ doesn't exist). Is it a problem for me?

**Comment by anarkiwi** [comment]
Apologies for missing your Q&A!

If you use Prometheus, then this shouldn't be a problem - only if you are logging direct from Gauge to InfluxDB, will this be a problem.
